.github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+name: CI-Pipeline
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  call-terraform-ci-pipeline:
+    uses: terraform-ibm-modules/common-dev-assets/.github/workflows/terraform-module-ci.yml@main
```

Comment:
Reusable workflows are no longer in common-dev-assets, we have moved them to common-pipeline-assets and pin them to a specific version. You can use this:
`terraform-ibm-modules/common-pipeline-assets/.github/workflows/common-terraform-module-ci.yml@v1.2.0`

Reply:
@toddgiguere - Ok this is coming from the module template at https://github.com/terraform-ibm-modules/terraform-ibm-module-template, so we should update there as well.

Reply:
My mistake actually - I took an older version of the template.